apps/simple-trading-app/project.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
chore (blocking): somewhere in here you will need to set a different port from default for testing purposes, as the test run in parallel if everything tries to start on the same port we get errors